virtcontainers/mount_test.go

```diff
@@ -8,14 +8,15 @@ package virtcontainers
 import (
 	"bytes"
 	"fmt"
-	"github.com/stretchr/testify/assert"
```

Comment:
How did that ever pass the fmt checks? :-/

Reply:
Well tbh I'm wondering how this entire test can have not failed under the CI until now as we should be launching the tests in the CI as a non-root user and allowing the tests to elevate privs to `root` and re-run.

Any thoughts on that @chavafg, @GabyCT, @amshinde?